src/components/LoginForm.jsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { login } from '../actions/actionCreators';
+
+const mapStateToProps = store => ({
+  games: store.games,
+});
+
+const mapDispatchToProps = dispatch => ({
+  signIn: (username, password) => {
+    dispatch(login(username, password));
+  },
+});
+class LoginForm extends React.Component {
```

Comment:
Hi David, 

Can we change this to a functional component?  Below is a link where you can reference on how to do this

https://daveceddia.com/redux-mapdispatchtoprops-object-form/

Reply:
I will refactor but I would like to get it working first